### ISSUE TITLE
increased/decreased oxidative stress 

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3394]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. increased oxidative stress
2. decreased oxidative stress 

>2. Synonyms (e.g., absent spleen)

1. elevated oxidative stress
2. reduced oxidative stress

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. A condition characterized by an increase in the accumulation of free radical groups in the body, which creates a potentially unstable and damaging cellular environment linked to tissue damage, accelerated aging, and degenerative disease.
2. A condition characterized by an decrease in the accumulation of free radical groups in the body, which relieves a potentially unstable and damaging cellular environment linked to tissue damage, accelerated aging, and degenerative disease.

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID: 32048631

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

oxidative stress (MP:0003674)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0003-3645-6803
--

**Comment by sbello** [comment]
We currently have the term "oxidative stress" (MP:0003674). It seems unclear to me how the new terms would relate to this. Possibly it would be better to use the susceptibility pattern? So increased/decreased susceptibility to oxidative stress with the definition related to some inducing agent that is expected to produce oxidative stress. 

It may also be useful to add and abnormal reactive oxygen species level parent to group these with the existing term.

**Comment by sbello** [comment]
Added decreased reactive oxygen species production (MP:0014607) since oxidative stress is already defined as accumulation of reactive oxygen species and has the synonym increased reactive oxygen species production
